oak_functions/wasm/src/lib.rs

```diff
@@ -463,7 +463,12 @@ where
     }
 
     pub fn handle_invoke(&self, request: Request) -> anyhow::Result<Response> {
-        let request_bytes = request.body;
+        let response_bytes = self.handle_raw_invoke(request.body)?;
+        Ok(Response::create(StatusCode::Success, response_bytes))
+    }
```

Comment:
Do you have any thoughts whether we plan to eventually align the the two runtimes response handling? I think there's good arguments against using the current runtime's proto wrapper in the UEFI runtime, but also eventual consistency would be nice.

Reply:
I think the plan is to eventually align the two runtimes once we have settled on an approach to framing and an IDL.